tests/likelihood_modules/test_likelihoods.py

```diff
@@ -1,45 +1,88 @@
 from pathlib import Path
+from typing import Union
 
 import numpy as np
 import pytest
 import torch
 
 from careamics.config.likelihood_model import GaussianLikelihoodModel, NMLikelihoodModel
-from careamics.config.nm_model import GaussianMixtureNmModel
+from careamics.config.nm_model import GaussianMixtureNmModel, NMModel
 from careamics.models.lvae.likelihoods import likelihood_factory
 from careamics.models.lvae.noise_models import noise_model_factory
 
 
-def test_gaussian_likelihood():
-    config = GaussianLikelihoodModel(model_type="GaussianLikelihoodModel")
+def create_dummy_noise_model(
```

Comment:
`create_dummy_noise_model` can be a fixture that is reused in both `test_likelihoods` and `test_noise_model`, could add a todo and do it later

Reply:
I'll be happy to sit down and see how to do that ;)